globals.json

```diff
@@ -496,6 +503,9 @@
 		"it": false,
 		"jasmine": false,
 		"pending": false,
+		"loadFixtures": false,
```

Comment:
jasmine itself doesn't have loadFixtures and setFixtures. So I will remove them from this PR.